src/itensor.jl

```diff
@@ -409,11 +485,13 @@ function summary(io::IO,
   print(io," \n",typeof(store(T)))
 end
 
+# TODO: make a specialized printing from Diag
+# that emphasizes the missing elements
```

Comment:
I like this idea 👍

Reply:
Julia has nice printing for their Diagonal matrix type:

```julia
julia> Diagonal([1,2,3])
3×3 Diagonal{Int64,Array{Int64,1}}:
 1  ⋅  ⋅
 ⋅  2  ⋅
 ⋅  ⋅  3
```

It could be nice to generalize that.